WebDriverAgentLib/FBAlert.m

```diff
@@ -147,6 +168,29 @@ - (BOOL)dismissWithError:(NSError **)error
   return [cancelButton fb_tapWithError:error];
 }
 
+- (BOOL)clickAlertButton:(NSString *)label withError:(NSError **)error {
+  
+  XCUIElement *alertElement = self.alertElement;
+  NSArray<XCUIElement *> *buttons = [alertElement descendantsMatchingType:XCUIElementTypeButton].allElementsBoundByIndex;
+  XCUIElement *requestedButton;
+  
+  for(XCUIElement *button in buttons) {
+    if([[button wdLabel] isEqualToString:label]){
```

Comment:
I'm not sure. I think most of WDA is currently case insensitive (at least when comparing on labels). Also, consider the - admittedly unlikely - events of having two labels with the same text but different casing. How would we handle that?